src/pycountry/tests/test_modify.py

```diff
@@ -0,0 +1,20 @@
+import pycountry
+
+
+def test_add_entry():
```

Comment:
This is where the `_clear_` method comes in really handy:
1. `_clear_` empties out the DB
2. When I then query it, the `@lazy_load` decorators load the DB again, i.e. I've got a "clean" version of the DB

This allowed me to formulate the two tests independently of each other.